### PR TITLE
SENTRY: sentry

### LINKED ISSUE #64: MONITORING: Configure Sentry for FE and BE

### PR DESCRIPTION
**Pull request by markrawls**
Closes #64

### CODE REVIEW

File: backend/backend/settings/__init__.py [hunk 1]
```diff
@@ -101,3 +104,23 @@
...
+        dsn=SENTRY_DSN,
+        integrations=[DjangoIntegration()],
+        send_default_pii=True,
+        environment=ENVIRONMENT,
+    )
+
+ZYGOAT_FRONTEND_META_CONFIG = {
+    "sentry_dsn": prod_required_env(
+        "DJANGO_FRONTEND_SENTRY_DSN",
+        default="https://8db6b816b01548e0bacdb9bdbeb2caf8@o39628.ingest.sentry.io/5339664",
```
Comment (ambikads) [inline backend/backend/settings/__init__.py:123]: Where does this default go to? the portunus frontend project in sentry has a different DSN.

File: frontend/pages/_app.js [hunk 2]
```diff
@@ -14,8 +17,18 @@ import ProtectedPage from '@@/components/ProtectedPage';
 import { ZENDESK_CHAT_KEY } from '@@/utils/constants/chat';
 import { googleAnalyticsEffect } from '@@/utils/google-analytics';
 
+import env from 'utils/env';
+
 import '@@/global.css';
 
+if (typeof window !== undefined) {
```
Comment (Ian-MacLeod) [inline frontend/pages/_app.js:24]: should be 'undefined'

File: backend/backend/settings/__init__.py (hunk 1 above)
Comment (markrawls) [reply backend/backend/settings/__init__.py:123]: I just double checked sentry and this is the right one, not sure which one you're seeing.

File: frontend/pages/_app.js (hunk 2 above)
Comment (markrawls) [reply frontend/pages/_app.js]: Whoops, my bad.

**Comment by markrawls** [comment]
Tests have to pass on it first, but sure

File: backend/backend/settings/__init__.py (hunk 1 above)
Comment (ambikads) [reply backend/backend/settings/__init__.py:121]: my bad

**Pull request merged**